src/Symfony/Component/Security/Guard/GuardAuthenticatorHandler.php

```diff
@@ -129,15 +130,22 @@ public function handleAuthenticationFailure(AuthenticationException $authenticat
         ));
     }
 
-    private function migrateSession(Request $request)
+    /**
+     * Call this method if your authentication token is stored to a session.
+     *
+     * @final since version 2.8
```

Comment:
"since version 2.8" should be removed. I don't think we are doing that anywhere else.

Reply:
We do when we want to hint that the `@final` has been added after the method has been introduced. i.e. when it's deprecated to extend from it.
Which is *not* the case here, so yes, this has to be removed.

Reply:
see 5c2b2bb2ce